contrib/prometheus/synapse-v2.rules

```diff
@@ -20,18 +20,18 @@ groups:
     expr: 'sum(synapse_http_server_request_count:by_method) by (servlet)'
```

Comment:
What is this file? Some kind of post-processing to have Prometheus compute and store derived metrics from the scraped metrics?

Reply:
Yeah, exactly :)